### DISCUSSION TITLE
Formatting the p style with separate css file - Font size doesn't change

### DISCUSSION THREAD

**Discussion by gauthiervasseur** [marp-team/marp#535] [Q&A] [answered]
Greetings,

While all the other styles (h1-h6, ol, ul...) see their size change, I just can't get regular text style to impact the font size.

This is the CSS file sample I'm testing things with: 

/**
 * @theme DWA
 * @auto-scaling fittingHeader,math
 */

@import 'default';

p {
  font-family: 'arial', serif;
  font-size: 100px;
}

The font type changes, not the size.

Any idea?

**Comment by yhatt** [comment]
The provided style looks working for me. `font-size: 100px` is applied to `<p>` elements correctly.

```markdown
---
theme: DWA
---

This is a paragraph
```

<img width="662" src="https://github.com/user-attachments/assets/63ce932a-b98c-415b-a56f-2ba03230957a">

You can inspect whether the style was applied style correctly, by using the developer tool on your browser to the converted HTML.

**Reply by gauthiervasseur** [reply, marked as answer]
Thank you for you quick reply on that one. I had a style over riding the CSS in the middle of this large deck!

On a side note, do you offer MARP/Markdown conferences or workshops to corporate org or universities (with honorarium)?
